src/wizard/hints.jl

```diff
@@ -66,7 +66,7 @@ function provide_hints(state::WizardState, path::AbstractString)
                 print(state.outs,   "    This file is likely input to CMake. ")
                 println(state.outs,   "The recommended options for CMake are")
                 println(state.outs)
-                printstyled(state.outs, "      cmake -DCMAKE_INSTALL_PREFIX=\$prefix -DCMAKE_TOOLCHAIN_FILE=/opt/\$target/\$target.toolchain", bold=true)
+                printstyled(state.outs, "      cmake -DCMAKE_INSTALL_PREFIX=\$prefix -DCMAKE_TOOLCHAIN_FILE=/opt/\$target/\$target.cmake", bold=true)
```

Comment:
Well spotted (and I even modified this yesterday...), but this should rather use the environment variable `${CMAKE_TARGET_TOOLCHAIN}`:
```suggestion
                printstyled(state.outs, "      cmake -DCMAKE_INSTALL_PREFIX=\$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}", bold=true)
```

Reply:
Confirmed working. Applied!